evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -811,6 +815,26 @@ contract FluxAggregator is AggregatorInterface, Owned {
     return payment.mul(oracleCount()).mul(RESERVE_ROUNDS);
   }
 
+  function addOracle(
+    address _oracle,
+    address _admin
+  )
+    private
+    onlyUnenabledAddress(_oracle)
+  {
+    require(_admin != address(0), "cannot set admin to 0");
+    require(oracles[_oracle].admin == address(0) || oracles[_oracle].admin == _admin, "owner cannot overwrite admin");
+
+    oracles[_oracle].startingRound = getStartingRound(_oracle);
+    oracles[_oracle].endingRound = ROUND_MAX;
+    oracleAddresses.push(_oracle);
+    oracles[_oracle].index = uint16(oracleAddresses.length.sub(1));
```

Comment:
Completely optional, but if we wanted to squeeze out a bit of gas savings when adding oracles, we could store the index at the length of oracleAddresses before pushing so that we don't have to call the SafeMath `.sub(1)` on the length.